server.js

```diff
@@ -1,27 +1,136 @@
+
+import express from "express"; 
 import cors from "cors";
 import mongoose from "mongoose";
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+// //////////////////////////////////////////////////////////////////////// //
+// //////////////////// CONNECTION TO DISPLAY ///////////////////////////// //
+// //////////////////////////////////////////////////////////////////////// //
+
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-happy-thoughts-api";
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
 mongoose.Promise = Promise;
 
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+// //////////////////////////////////////////////////////////////////////// //
+// //////////////////////////// SCHEMA //////////////////////////////////// //
+// //////////////////////////////////////////////////////////////////////// //
+
+const ThoughtsListSchema = new mongoose.Schema({
+  message: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 140,
+    trim: true
+  },
+  likes: {
+    type: Number,
+    default: 0,
+  },
+  createdAt: {
+    type: Date,
+    default: Date.now
+  }
+}); 
+
+const ThoughtsList = mongoose.model("ThoughtsList", ThoughtsListSchema);
+
+// //////////////////////////////////////////////////////////////////////// //
+// //////////////////////////// STARTPAGE ///////////////////////////////// //
+// //////////////////////////////////////////////////////////////////////// //
+
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send({
+    "Hello": "Welcome to this Happy Thought API!",
+    "Endpoints": [
+    {"/": "Startpage / Api Info"},
+    {"/thoughts": "GET. See 20 thoughts in descending order"},
+    {"/thoughts": "POST. Post your thoughts"},
+    {"/thoughts/:thoughtId/like": "POST. Update likes"}
+    ]
+    });
+});
+
+// ///////////////////////////////////////////////////////////////////// //
+// //////////////////////////// GET //////////////////////////////////// //
+// ///////////////////////////////////////////////////////////////////// //
+
+app.get("/thoughts", async (req, res) => {
+  const displayedThoughts = await ThoughtsList.find().sort({createdAt: 'desc'}).limit(20).exec();
+    try {
+      res.status(200).json({
+        success: true, 
+        response: displayedThoughts,
+        message: "Successfully found thoughts"
+      });
+  } catch(e) {
+    res.status(400).json({
+      success: false,
+      response: e,
+      message: "Did not find thoughts", 
+      error: e.errors
+     });
+  }
+});
```

Comment:
Nice job creating a catch error on this one, something I need to update mine with!